src/com/facebook/buck/swift/SwiftCompile.java

```diff
@@ -125,7 +125,10 @@
     this.objectPath = outputPath.resolve(escapedModuleName + ".o");
 
     this.srcs = ImmutableSortedSet.copyOf(srcs);
-    this.compilerFlags = compilerFlags;
+    ImmutableList.Builder<String> compilerFlagsBuilder = ImmutableList.builder();
+    compilerFlagsBuilder.addAll(swiftBuckConfig.getFlags().orElse(ImmutableSet.of()));
```

Comment:
Why not just make SwiftBuckConfig#getFlags always just return a (maybe empty) ImmutableList?

Then I'd slightly prefer this to look like:

```java
this.compilerFlags = new ImmutableList.Builder<String>()
    .addAll(swiftBuckConfig.getFlags())
    .addAll(compilerFlags)
    .build();
```

or something along those lines.